airbyte-integrations/connectors/source-clickup-api/source_clickup_api/clickup_api.yaml

```diff
@@ -0,0 +1,54 @@
+version: "0.1.0"
+
+definitions:
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_clickup_api/schemas/{{ options['name'] }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer: []
+  requester:
+    type: HttpRequester
+    name: "{{ options['name'] }}"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "Authorization"
+      api_token: "{{ config['api_token'] }}"
+  retriever:
+    type: SimpleRetriever
+    $options:
+      url_base: "https://api.clickup.com/api/v2"
+    name: "{{ options['name'] }}"
+    primary_key: "{{ options['primary_key'] }}"
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+
+streams:
+  - type: DeclarativeStream
```

Comment:
@josix are you planning to add more streams? (Tasks, Lists, etc?)

Reply:
Yeah, I'll add Tasks stream these days.